tests/integration/test_issues.py

```diff
@@ -4,34 +4,35 @@
 import pytest
 
 from rat.model import Model
+from rat import optimize
 
 test_dir = pathlib.Path(__file__).parent
-issue_data_dir = os.path.join(test_dir, "issue_data")
+issue_data_dir = test_dir / "issue_data"
```

Comment:
Check out this weird path join [syntax](https://docs.python.org/3/library/pathlib.html#operators). I do not like it XD

Reply:
looool, agreed, until everyone is doing it and I get used to it, os.path.join ftw

Reply:
But to be clear, this is just a few tests, whatever is fine, no need to go change this.

Reply:
Agreed